workers/data_refinery_workers/downloaders/utils.py

```diff
@@ -52,6 +52,14 @@ def end_downloader_job(job: DownloaderJob, success: bool):
     if success:
         logger.info("Downloader Job completed successfully.",
                     downloader_job=job.id)
+    else:
+        logger.info("Downloader job failed!",
+                    downloader_job=job.id,
+                    downloader_task=job.downloader_task)
+        if not job.failure_reason:
```

Comment:
Can this be refactored such that if the failure reason is present, it gets passed to the logger?